db/migrations/000015-update_posts_table_for_erc721.sql

```diff
@@ -0,0 +1,17 @@
+-- +migrate Up
```

Comment:
We still need to index these fields in the worker queue?

Reply:
Yup.. still WIP